docs/source/technical_tutorials/working_with_r/cmr_search_in_r.ipynb

```diff
@@ -15,7 +15,7 @@
    "source": [
```

Comment:
<p>You made a table you should use it.</p>

<pre class="ql-syntax" spellcheck="false">collection_id = unlist(df[2, "id"])
collection_id
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/495/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>